README.md

```diff
@@ -1,37 +1,71 @@
+
+1. Face recognition using iOS Vision API
+2. Classify cropped face using IBM Visual Recognition
+3. Get details from IBM Cloudant using the classification
+4. Overlay the data in front of the user in the mobile camera view
+
+
+# Included Components
+1. Swift mobile app
+    a. Face Recognition using Vision API
+    b. ARKit : An iOS augmented reality platform        
+2. IBM Visual Recognition: An IBM service to analyze the visual content of images or video frames to understand what is happening in a scene
+3. IBM Cloudant DB: A highly scalable and performant JSON database service
 
-![ARResume Architecture](https://github.com/sanjeevghimire/ARBasedResumeWithFaceRecognition/blob/master/ARResume.jpg "ARResume Architecture")
 
 # Steps
 
-1. git clone 
-2. Get IBM Visual Recognition and update the Credentials.swift
+1. Clone the project using `git clone https://github.com/IBM/ARBasedResumeWithFaceRecognition`
+2. Login to IBM Bluemix account, create a IBM Visual Recognition service and save the credentials.
+3. Using the IBM Visual Recognition Beta Tool, create a classifier. A classifier will train the visual recognition so that the    VR API will recognizse different images of the same person. Use atleast 10 images of your head shot and also use a negative    training by using a different headshot which is not you.
+4. Create a IBM Cloudant database and save the credentials. Create a JSON with the classificationId as one of the key-value      pair. This ID will be used to retrieve details about the classified person.
+5. Update the `credentials.swift` file with the Cloudant and VR credentials as the app communicates with both of the services    using their API
+6. Open the cloned project using `Xcode`
+7. From command line, run `pod install`
+8. After `step 7` is done, run `carthage update --platform iOS`
+9. Once `step 7` and `step 8` are successful, `build and run` using Xcode.
+
+
 3. Get IBM Cloudant Database and update the Credentials.swift
 4. Open the project using Xcode
 5. Build and Run
 
 # Test
-To test you can do the following
+To test locally you can do the following: 
 1. Train IBM visual recognition with your head shot images. Please use atleast 10 images of your head shot and use a negative trainig by using a different head shot which is not you.
 2. Update the cloudant database using the classification id from the training to store your information. The JSON will be in the format:
 
 ```
-{
-  "_id": "c2554847ec99e05ffa8122994f1f1cb4",
-  "_rev": "3-d69a8b26c103a048b5e366c4a6dbeed7",
-  "classificationId": "SanjeevGhimire_334732802",
-  "fullname": "Sanjeev Ghimire",
-  "linkedin": "https://www.linkedin.com/in/sanjeev-ghimire-8534854/",
-  "twitter": "https://twitter.com/sanjeevghimire",
-  "facebook": "https://www.facebook.com/sanjeev.ghimire",
-  "phone": "1-859-684-7931",
-  "location": "Austin, TX"
+{  
```

Comment:
We need to explain how a user can update the database, i guess a curl call is the easiest way?